orttraining/orttraining/test/model/training_runner_test.cc

```diff
@@ -60,6 +60,56 @@ TEST(TrainingRunnerTest, Basic) {
   ASSERT_TRUE(status.IsOK()) << status.ErrorMessage();
 }
 
+// This test verifies a simple pipeline run with 3 partitions.
+// TODO: enable this test after distributed run is not enabled in CI.
```

Comment:
>not [](start = 51, length = 3)

remove "not"? BTW how is OpenMPI enabled for this test? Changing build.py?

Reply:
Currently I am not sure how the distributed CI would run the test -- either through separate independent app (like the nightly build does) or the gtest suite. For either case, we have some test that is ready to be added once it is up. If it turns out that the gtest suite cannot run at that point, we can remove this test or convert it into another separate exe.